awx/api/urls/urls.py

```diff
@@ -149,3 +149,7 @@
     from awx.api.swagger import SwaggerSchemaView
 
     urlpatterns += [re_path(r'^swagger/$', SwaggerSchemaView.as_view(), name='swagger_view')]
+
+    from awx.api.urls.debug import urls as debug_urls
+
+    urlpatterns += [re_path(r'^debug/', include(debug_urls))]
```

Comment:
Should this be conditional? See 

https://github.com/ansible/awx/blob/fd671ecc9d49845f1e0fa09ec178d1d6518061fe/awx/main/utils/filters.py#L370-L371

Reply:
I'm in this same block as the swagger api that has the test:
```
if settings.SETTINGS_MODULE == 'awx.settings.development':
```

but I could switch that to 
````
from awx import MODE
....
 if MODE == 'development':
```